decoder_event.go

```diff
@@ -421,12 +421,16 @@ func (p *Decoder) parseMetaMsg(e *Event) (nextChunkType, bool, error) {
 			}
 
 			// key
+<<<<<<< HEAD
```

Comment:
oops :)

Reply:
thanks @glaslos for reviving that old PR I totally forgot about. It looks like there was a little merge problem.